roles/maas_pacemaker/tasks/main.yaml

```diff
@@ -113,3 +113,4 @@
       - "{{ maas_pgsql_check_port|default(23267) }}"
     maas_pg_udp_ports:
       - 5405
+  when: enable_firewall | bool
```

Comment:
I think all of these can just be `enable_firewall` since that itself is a boolean (also for the other changes)

Reply:
Good point